### ISSUE TITLE
Tab completion not working for the tutorial 2022

### ISSUE THREAD

**Issue by giovannipizzi** [aiidateam/aiida-tutorials#451]
Probably this is more of an issue for the deployment, so feel free to move it.

Tab completion does not work for me in the terminal.

It starts working if I run
```
eval "$(_VERDI_COMPLETE=bash_source verdi)"
```

We should probably add this line to the ~/.bashrc or similar way to enable it by default

**Comment by mbercx** [comment]
True, I missed this one in adapting the deployment. I had quite a few issues in my attempts to adapt the built image, unfortunately, probably due to my inexperience. I can try to see if I can add it, but it might be risky to try and adapt the image right now.

Maybe it makes more sense to simply include these instructions on the tutorial page for now.
